### PR TITLE
Move the implicit imports from plugin to yaml

### PR DESCRIPTION
**Pull request by Fire-Cube**
- move the implicit imports from ImplicitImport.py to the yaml files
- fix schema
- add schema-check for stdlib2 and stdlib3

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -275,6 +275,6 @@
     "esbonio.sphinx.confDir": "",
     "cSpell.spellCheckOnlyWorkspaceFiles": true,
     "yaml.schemas": {
-        "misc/nuitka-package-config-schema.json": "standard.nuitka-package.config.yml"
+        "misc/nuitka-package-config-schema.json": ["standard.nuitka-package.config.yml", "stdlib2.standard.nuitka-package.config.yml", "stdlib3.standard.nuitka-package.config.yml"]
```
Comment (kayhayen) [inline .vscode/settings.json:278]: merge problem?

Comment (Fire-Cube) [reply .vscode/settings.json:278]: Why do you think?

File: nuitka/plugins/standard/standard.nuitka-package.config.yml [hunk 2]
```diff
@@ -110,6 +174,10 @@
     empty_dirs:
       - "."
 
+- module-name: "curses"
+  implicit-imports:
+    - depends:
+        - "_curses"
```
Comment (kayhayen) [inline nuitka/plugins/standard/standard.nuitka-package.config.yml:180]: That's actually a stdlib module that we overlooked before.

File: nuitka/plugins/standard/standard.nuitka-package.config.yml [hunk 3]
```diff
@@ -130,6 +198,19 @@
   implicit-imports:
     - depends:
         - "cv2.cv2"
+        - "numpy"
+        - "numpy.core"
+
```
Comment (kayhayen) [inline nuitka/plugins/standard/standard.nuitka-package.config.yml:203]: I will try and tweak auto-format for Yaml to enfore a separation of lines, probably on the ASCII level...

File: nuitka/plugins/standard/standard.nuitka-package.config.yml [hunk 4]
```diff
@@ -420,6 +867,32 @@
...
+        - "numpy.random.bit_generator"
+        - "numpy.random.bounded_integers"
+        - "numpy.random.common"
+        - "numpy.random.generator"
+        - "numpy.random.mt19937"
+        - "numpy.random.pcg64"
+        - "numpy.random.philox"
+        - "numpy.random.sfc64"
+
+      # TODO: Clarify if entropy is needed for 1.18 or at all.
```
Comment (kayhayen) [inline nuitka/plugins/standard/standard.nuitka-package.config.yml:893]: Thanks for even maintaining the TODOs properly, looks very good to my eyes.

**Review by kayhayen — APPROVED** [review]
This is extremely well done. For my constant scare of breaking things, I will write a small tool, that will iterate over all available modules, and produce an output, that I can then diff. In this instance, it will have to be identical. There is already code to iterate over all modules, and we basically only need to output plugin decisions. I will merge, once I have the tool written, hopefully early this week.

File: nuitka/plugins/standard/standard.nuitka-package.config.yml (hunk 2 above)
Comment (kayhayen) [reply nuitka/plugins/standard/standard.nuitka-package.config.yml:180]: I think on Windows, curses doesn't work until something from pip is installed, that kind of messes things up.

**Pull request merged**